src/cassandra/cli/load.cdl

```diff
@@ -0,0 +1,180 @@
+-- this cdl schema create the main tables for blueflood in cassandra using the cqlsh client
+-- usage:
+--   $ cqlsh hostname -f load.cdl -u username -p password
+-- example:
+--   $ cqlsh 127.0.0.1 -f ./blueflood/src/cassandra/cli/load.cdl
+
+CREATE KEYSPACE IF NOT EXISTS "DATA"
+    WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };
+
+
+CREATE TABLE IF NOT EXISTS "DATA".metrics_locator (
+    key bigint,
+    column1 text,
+    value text,
+    PRIMARY KEY (key, column1)
+) WITH COMPACT STORAGE AND speculative_retry = 'NONE';
```

Comment:
Maybe i should completely eliminate these other two settings two for CREATE TABLE?

They are explicitly different from the default that looks like it might be better for local (single node i.e. your computer) testing.